code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -1,42 +1,28 @@
 /datum/gear/accessory
-	display_name = "armband, red"
-	path = /obj/item/clothing/accessory/armband
```

Comment:
code/modules/client/preference_setup/loadout/loadout_accessories.dm:3:error: /obj/item/accessory/: undefined type path

Compile error

Reply:
it stopped working again save me from this hell